app/delivery/send_to_providers.py

```diff
@@ -65,8 +66,24 @@ def send_sms_to_provider(notification):
                 # providers as a slow down of our providers can cause us to run out of DB connections
                 # Therefore we pull all the data from our DB models into `send_sms_kwargs`now before
                 # closing the session (as otherwise it would be reopened immediately)
+
+                # We start by trying to get the phone number from a job in s3.  If we fail, we assume
+                # the phone number is for the verification code on login, which is not a job.
+                my_phone = None
+                try:
+                    my_phone = get_phone_number_from_s3(
+                        notification.service_id,
+                        notification.job_id,
+                        notification.job_row_number,
+                    )
+                except BaseException:
+                    my_phone = redis_store.get(f"2facode_{notification.id}")
+                    if my_phone:
+                        my_phone = my_phone.decode("utf-8")
```

Comment:
We don't want this phone number though, right, because it's the verification number?  What happens if we retrieve this number instead?

Reply:
No, we need this phone number.  When you sign in, you receive a verification code on your phone as part of the process.  This is that notification.

I guess it will go away when login.gov is in place.